app/tests/services/sql.tests.js

```diff
@@ -0,0 +1,45 @@
+describe('SQL', function() {
+  'use strict';
+
+  var baseURI;
+
+  beforeEach(angular.mock.module('sql'));
+
+  beforeEach(function() {
+    angular.mock.inject(function($injector) {
+      baseURI = $injector.get('baseURI');
+    });
+  });
+
+  describe('SQL baseURI', function() {
+    it('base uri should keep path', inject(function(baseURI) {
+      //retrieve initial path
+      expect(baseURI.getURI('/test')).toEqual(window.location.protocol + '//' + window.location.host + window.location.pathname+ '/test');
```

Comment:
as it's a test, isn't it possible to compare it with a hardcoded string like `http://example.com/test`. That would make the test more readable.